src/fides/api/ctl/routes/health.py

```diff
@@ -72,7 +72,11 @@ async def health(
     db: Session = Depends(get_db),
 ) -> Dict:  # Intentionally injecting the ops get_db
     """Confirm that the API is running and healthy."""
-    database_health = get_db_health(CONFIG.database.sync_database_uri, db)
+    database_health = (
+        get_db_health(CONFIG.database.sync_database_uri, db=db)
+        if CONFIG.database.sync_database_uri is not None
+        else "No database uri provided"
```

Comment:
given the way that the database URIs are computed in the configs, I think its better to fix the type there than put this _everywhere_

It's not actually possible for these to be empty anyway, it's just a janky fix because they're getting defined at runtime. The alternative is just to put in a default and remove the `optional`

Reply:
We used to use a default on the ops side, but removed it because we didn't want things defaulting to `localhost` without someone explicitly setting it to that. I think I could still remove `optional` in the settings model and have the validator handle the case where it is missing.

If no value is present nothing is going to work so I think it is actually more clear to remove the optional anyway. Does this make sense to you if I get Pydanic happy with it?

Reply:
totally! I think it is ok to have it default to localhost, I'm not sure what the issue with that would be